app/api/events.py

```diff
@@ -120,6 +121,12 @@ def after_create_object(self, event, data, view_kwargs):
                                  status='accepted')
         save_to_db(role_invite, 'Organiser Role Invite Added')
         if data.get('original_image_url'):
+            try:
+                img = urllib.request.urlopen(data['original_image_url']).read()
```

Comment:
Normally, I wouldn't say anything, but a try catch block of an image requestal should have it's own function if it's going to be repeated more than once. Encapsulate the try catch block in a function of it's own.